<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>column_axial_force</class>
 <widget class="QDialog" name="column_axial_force">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>94</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Column Axial Forces</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="group_checkbox">
       <property name="text">
        <string>Create Group</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="group_name">
       <property name="text">
        <string>0.3*Ag*fc</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="select_all">
     <property name="text">
      <string>Select All</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="limit_spinbox">
     <property name="prefix">
      <string/>
     </property>
     <property name="suffix">
      <string> * Ag * fc</string>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.300000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
